css/style.css

```diff
@@ -16,4 +16,138 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/* Navbar Styling */
+#navbar {
+    background: #fff;
+    color: #000;
+    overflow: auto;
+    justify-content: space-around;
+  }
 
+#navbar img{
+        width: 70px;
```

Comment:
Would you please set the width and height of the image in the navbar to % than px,

Reply:
@Mathias02 I used PX because it is responsive even if I didn't make the website more responsive, but it is responsive with the navbar

Reply:
![Screenshot_2022-06-16_12-57-43](https://user-images.githubusercontent.com/45645052/174056584-d11a6b14-8d64-45a0-8713-9e3ba4bcaf22.png) us this link https://62ab0acbf8ac9c705bad03a6--unrivaled-panda-b57cb1.netlify.app/ to test it using the developer tools